recipes/plant_isce3/LICENSE

```diff
@@ -0,0 +1,210 @@
+Copyright 2022, by the California Institute of Technology. ALL RIGHTS RESERVED. United States Government Sponsorship acknowledged. Any commercial use must be negotiated with the Office of Technology Transfer at the California Institute of Technology.
```

Comment:
This file is in the tarball. There is no need to re-add it here.

Reply:
Please remove it. It is already in the tarball and conda will pick it up.

Reply:
Sorry for that, just removed the file. Thank you, @ocefpaf.